modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Sidebar/TasksSidebar/index.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import classNames from 'classnames';
+import {Fragment} from 'react';
+
+import {useSidebarTask} from '../../../hooks/useSidebarTask';
+import i18n from '../../../i18n';
+import {TestraySubTask, TestrayTask} from '../../../services/rest';
+import {StatusesProgressScore, chartClassNames} from '../../../util/constants';
+import TaskbarProgress from '../../ProgressBar/TaskbarProgress';
+
+type QuantityTaskBadgeProps = {
+	classStyle?: string;
+	count: number;
+};
+
+type TaskSidebarProps = {
+	expanded: boolean;
+};
+
+type SubtaskCardProps = {
+	expanded: boolean;
+	subtask: {
+		name: string;
+		score: number;
+	};
+};
+
+const QuantityTaskBadge: React.FC<QuantityTaskBadgeProps> = ({
+	classStyle,
+	count,
+}) => {
+	return (
+		<div
+			className={classNames(
+				`align-items-center d-flex justify-content-center quantity-task-badge `,
```

Comment:
Put single quotes (' ')
```suggestion
				'align-items-center d-flex justify-content-center quantity-task-badge ',
```